src/modules/site-v2/templates/errors/400.html

```diff
@@ -5,5 +5,5 @@
 {% block content %}
   <h1>Internal Server Error :(</h1>
   <h3>We're having some problems</h3>
-  <h3>Instead, look at all these <a href="{{ url_for('request_strains.strains') }}">worms</a>!</h3>
+  <h3>Instead, look at all these <a href="{{ url_for('request_strains.request_strains') }}">worms</a>!</h3>

```

Comment:
I discussed with Erik and he said it makes sense to link to the homepage on the error pages. I updated [here](https://github.com/AndersenLab/CAENDR/commit/d3c29c5cec04dac0ece8fbe62bbcfa1345bbe8ba).